recipe/meta.yaml

```diff
@@ -1,9 +1,9 @@
 package:
   name: gmt
-  version: 6.2.0.dev12+922af84
+  version: 6.2.0rc1
```

Comment:
Note that unlike the regular `dev` versions, there is no dot `.` between the version string `6.2.0` and `rc1` according to https://github.com/conda-forge/cfep/blob/master/cfep-05.md#appendix.